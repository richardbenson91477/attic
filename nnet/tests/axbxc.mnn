3 3 1
0.01 0.1 0.3 100.0 90000 1
axbxc.mni
