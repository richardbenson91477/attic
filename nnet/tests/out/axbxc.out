0.009525 
0.904842 
0.902942 
0.051563 
0.902629 
0.057064 
0.064631 
0.758208 
